.github/workflows/back-end.yml

```diff
@@ -0,0 +1,228 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: "Back-end"
+
+on:
+    pull_request: null
+    push:
+        branches:
+        - "main"
+
+permissions:
+    contents: "read"
+
+concurrency:
+    group: "${{ github.workflow }}-${{ github.ref }}"
+    cancel-in-progress: true
+
+jobs:
+    byte_level:
+        name: "0️⃣ Byte-level"
+        runs-on: "ubuntu-22.04"
+        timeout-minutes: 1
+        steps:
+        -
+            name: "Checkout repository"
+            uses: "actions/checkout@v3"
+        -
+            name: "Check file permissions"
+            run: |
+                test "$(find ./ -type f -not -path './.git/*' -executable)" = ""
```

Comment:
When badly configured git clients/Windows user turn on the execute permission on files.
`chmod +x`